contrib/ansible/vagrant/provision.sh

```diff
@@ -0,0 +1,9 @@
+echo "hello, here is  a sample provisioning script that demonstrates everything works"
+ls /vagrant
+### Required for ansible??
+### Otherwise 'no yum module installed'...
+sudo yum install -y yum # needed for the ansible python stuff
+sudo yum install -y docker
+sudo yum install -y openssl # needed for making certs
```

Comment:
why do we need openssl to make certs? I take it easy rsa uses it under the covers?  /me has not looked into easy rsa.